packages/vue/src/components/FSelectField/FSelectField.spec.ts

```diff
@@ -183,6 +183,22 @@ describe("events", () => {
         expect(vModelValue).toEqual({ id: 1, fruit: "banana" });
     });
 
+    it("should support v-model by emitting update:modelValue event with null", async () => {
+        const wrapper = mount(
+            createTestComponentWithOptions([
+                { text: "BananaObject", value: { id: 1, fruit: "banana" } },
+                { text: "AppleObject", value: { id: 2, fruit: "apple" } },
+            ]),
+            { props: { modelValue: null } },
+        );
+        await wrapper.vm.$nextTick();
+        const vModelValue = wrapper
+            .findComponent(FSelectField)
```

Comment:
```suggestion
            .getComponent(FSelectField)
```

Föredra `getComponent()` över `findComponent()` när vi förväntar oss att den ska hitta elementet/komponenten. Motivering: då smäller testet direkt här med ett bra felmeddelande istället.